1-3.md

```diff
@@ -0,0 +1,164 @@
+
+## Learning Outcomes
+
+Students should be able to:
+* Identify accessibility as a driving force for innovation in products
+* Recognize accessibility as a discipline that promotes  inclusion
+* Explain how accessibility can extend market reach for organizations
+* Identify how to minimize legal risk for some organizations
+
+## Competencies
+
+The following is a list of skills required for this unit.
+
+Studentss
+: Basic &mdash; Basic understanding of computers and the Web.
+
+Instructors
+: Medium &mdash; Good understanding of accessibility and inclusion. Instructors should have experience in the types of business benefits resulting from accessibility.
+
+{% comment %}
+
+## Duration
+
+Approximately 1-2 hours depending on demonstrated examples.
+
+{% endcomment %}
+
+## Topics to Teach
+
+The following topics help achieve the learning outcomes. Adapt the sequence of topics, the activities you do, and the homework you assign for your particular students and context.
+
+{% include excol.html type="all" %}
+
+{% include excol.html type="start" %}
+
+### Topic: Accessibility as a Drive for Innovation
+
+{% include excol.html type="middle" %}
+
+Explain how accessibility helps organizations remove architectural, digital, and social barriers. Use the resource [Business Case for Digital Accessibility (Drive Innovation)](https://www.w3.org/WAI/business-case/#drive-innovation).
+
+### Learning Outcomes
+
+* Identify the common features of accessibility and usability to deliver a more intuitive experience.
+* Explain how some innovative products which were originally developed for people with disabilities, are now used by all
+* Identify how accessibility contributes for more human-centered design interaction patterns.
+
+{% comment %}
+
+#### Duration
+
+Approximately 30 to 45 minutes.
+
+{% endcomment %}
+
+#### Teaching Ideas
+
+* Discuss how accessibility solutions are also enjoyed by many people regardless of their disability. Emphasize how many solutions originally thought for accessibility have become mainstream, such as the type-writer, telephone, punch cards, text to speech, email, and voice controls.
+* Promote discussion among students about which aspects of accessibility benefit them as well as people with disabilities. For instance, ask them which products they can use without looking at them or without pressing any button.
+
+#### Homework Ideas
+
+* Ask students to research how many people with disabilities there are in their organization. Emphasize that statistics on this matter are difficult to find and interpret. The concept of disability, the way it is measured in different cultures, and the right for people not to disclose their disabilities may influence the results.
```

Comment:
Hmmm - could be a privacy concern going around and asking people to disclose if they have a disability. Maybe "ask students to think of examples of people with disabilities they know of" or such?